pretix_servicefees/views.py

```diff
@@ -41,6 +41,10 @@ class ServiceFeeSettingsForm(SettingsForm):
         label=_('Do not charge per-ticket service fee on non-admission products'),
         required=False
     )
+    service_fee_skip_free = forms.BooleanField(
+        label=_('Do not charge per-ticket service fee on free products UNLESS the entire order is free'),
```

Comment:
Is the „unless“-part true? I read the code a bit different: if we charge a per-ticket service fee even on free products, it is only charged if the entire order is not free. So IMHO it is kind of the other way around: „Do not charge per-ticket service fee on free products – per-ticket service fees on free products are only charged if the entire order is not free“. Or shorter: „Do not charge per-ticket service fee on free products. Please note: if the entire order is free, no fees will be charged whatsoever.“